resources/scripts.js

```diff
@@ -125,112 +181,296 @@ function buttonPosition(loc) {
             onButton.style.left = '1235px';
             break;
     }
+    return;
 }
 
-function playerWin() {
-    displayMessage('You win!');
-    playerMoney += (betAmount * 2);
-    displayMoney.innerHTML = playerMoney;
-    buttonPosition(1);
-    betAmount = 0;
-    rollButtonState(false);
-    betButtonState(true);
+function chipChange(loc, color) {
+    switch (loc) {
+        case 1:
+            chipContainerElement.style.visibility = 'hidden';
+            break;
+        case 2:
+            chipContainerElement.style.visibility = 'visible';
+            break;
+    }
+    return;
 }
 
-function playerLose() {
-    displayMessage('You lose')
+function playerWin(type, amount) {
+    if (type === SECONDARY) {
+        lastPlayerMoney = playerMoney;
+        playerMoney += (amount * 2);
+        displayMessage(SECONDARYWIN);
+        moneyChange(0);
+        return;
+    }
+    displayMessage(PRIMARYWIN);
+    lastPlayerMoney = playerMoney;
+    playerMoney += (amount * 2);
+    pushSecondaryBets();
+    resetGame();
+    moneyChange(0);
+    return;
+}
+
+function playerLose(type) {
+    if (type === SECONDARY) {
+        lastPlayerMoney = playerMoney;
+        displayMessage(SECONDARYLOSE);
+        moneyChange(0);
+        return;
+    }
+    displayMessage(PRIMARYLOSE)
     if (playerMoney === 0) {
         gameOver();
         return;
     }
-    buttonPosition(1);
-    betAmount = 0;
-    rollButtonState(false);
-    betButtonState(true);
+    buttonPosition(playerRoll);
+    resetGame();
+    moneyChange(0);
+    return;
 }
 
-function placeBet() {
-    betAmount = betDialog.value;
+function makeBet() {
+    betAmount = betDialogElement.value;
     betAmount = parseInt(betAmount);
-    if (validateBet() === false) {
+    if (validateBet(betAmount) === false) {
         return;
     }
+    lastPlayerMoney = playerMoney;
     playerMoney -= betAmount;
-    displayMoney.innerHTML = playerMoney;
-    rollButtonState(true);
-    betButtonState(false);
+    moneyChange(betAmount);
+    buttonStates();
+    return;
+}
+
+function makeCome() {
+    let comeAmount = betDialogElement.value;
+    comeAmount = parseInt(comeAmount);
+    if (validateBet(comeAmount) === false) {
+        return;
+    }
+    lastPlayerMoney = playerMoney;
+    playerMoney -= comeAmount;
+    secondaryBetList.push(new SecondaryBet(1, comeAmount));
+    moneyChange(comeAmount);
+    betDialogElement.disabled = true;
+    comeButtonElement.disabled = true;
+    return;
+}
+
+function makePlace(num) {
+    // Error found:
+    // When making a second place bet, first bet is repeated
```

Comment:
![Screenshot_2024-09-07_at_14 48 42](https://github.com/user-attachments/assets/1a0cdcaa-3645-4c47-93ce-682fa59b248a)

Not sure the issue here: The first **place** bet works normally but when a second bet is made, it is repeated (with the new amount). Multiple **come** bets appear to work normally.

Reply:
Issue is that adding eventlisteners in functions will end up making duplicates. Will make this into global eventlistener.